src/Compilers/CSharp/Portable/BoundTree/Expression.cs

```diff
@@ -384,17 +447,92 @@ ImmutableArray<IMemberInitializer> IObjectCreationExpression.MemberInitializers
         {
             get
             {
-                BoundExpression initializer = this.InitializerExpressionOpt;
-                if (initializer != null)
-                {
-                    // ZZZ What's the representation in bound trees?
-                }
-
-                return ImmutableArray.Create<IMemberInitializer>();
+                var initializers = s_memberInitializersMappings.GetValue(this,
+                    objCreationExp =>
+                    {
+                        BoundExpression initializer = this.InitializerExpressionOpt;
+                        if (initializer != null)
+                        {
+                            var objInitializerExp = initializer as BoundObjectInitializerExpression;
```

Comment:
Personally, I prefer names to include full words rather than abbreviations. Feel free to ignore this for the time being.